### PR TITLE
Fix weird printing outputs, add more tests for calculate command

### LINKED ISSUE #109: [PE-D] Message not being correctly shown on Windows PowerShell
Message does not show correctly in Windows PowerShell:

Intended message as presented in user guide:

Perhaps the team can consider alternative methods to display the message such that it appears on Windows PS.

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: laiisaac/ped#3

### LINKED ISSUE #112: [PE-D] Characters not showing up properly
May want to use a different character for the breakdown of expenses, nonetheless not a big issue, honestly liked the use of a breakdown of expenses.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: LDerpy/ped#4

### LINKED ISSUE #128: [PE-D] Bar Appeared as An Invalid syntax
The bar in my screen.

The bar in the user guide

The bar appeared on my side looks like an invalid syntax so i believe the library or code for the bar  doesn't work on my computer.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: lihao-InfoSec/ped#7

### LINKED ISSUE #132: [PE-D] calculate command output invalid
After adding a single expenses and running the `calculate` command, the output given contains ??????? as seen below.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: siewyangzhi/ped#11

### PR DESCRIPTION
**Pull request by limjierui**
Fixes #132  Fixes #128  Fixes #112  Fixes #109 

Fix weird printing outputs

Add more tests for calculate command

### CODE REVIEW

File: src/test/java/seedu/mindmymoney/command/CalculateInputCommandTest.java [hunk 1]
```diff
@@ -23,66 +23,97 @@ public void setUp() {
...
+     * Asserts if user is able to calculate expenditure by date.
      */
     @Test
-    void calculateInputCommand_oneInput_expectCorrectOutput() throws MindMyMoneyException {
+    void calculateInputCommand_calculateByDate_expectCorrectOutput() throws MindMyMoneyException {
         ExpenditureList expenditureTestList = new ExpenditureList();
         CreditCardList creditCardTestList = new CreditCardList();
         IncomeList incomeList = new IncomeList();
         User user = new User(expenditureTestList, creditCardTestList, incomeList);
         String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```
Comment (khseah) [inline src/test/java/seedu/mindmymoney/command/CalculateInputCommandTest.java:34]: Maybe can add a new line here to "break up" the code, so its not a continuous chunk of code
```suggestion
        
        String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```

File: src/test/java/seedu/mindmymoney/command/CalculateInputCommandTest.java [hunk 2]
```diff
@@ -23,66 +23,97 @@ public void setUp() {
...
+    /**
+     * Asserts if user is able to calculate expenditure by month.
+     */
+    @Test
+    void calculateInputCommand_calculateByMonth_expectCorrectOutput() throws MindMyMoneyException {
+        ExpenditureList expenditureTestList = new ExpenditureList();
+        CreditCardList creditCardTestList = new CreditCardList();
+        IncomeList incomeList = new IncomeList();
+        User user = new User(expenditureTestList, creditCardTestList, incomeList);
+        String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```
Comment (khseah) [inline src/test/java/seedu/mindmymoney/command/CalculateInputCommandTest.java:69]: Similar as above
```suggestion
        
        String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```

**Review by khseah — COMMENTED** [review]
Nice job!

**Review by SeanHoWB — COMMENTED** [review]
looks good, after handling kh comments can merge liao

**Pull request merged**